installer/Migration/V5_2_0/lang-string/general.yaml

```
@@ -38,3 +38,15 @@ langStrings:
   -
     value: 'Enter a valid port number between {minValue} to {maxValue}'
     unitId: enter_valid_port_between_a_to_b
+  -
+    value: 'Time at Work'
+    unitId: time_at_work
+  -
+    value: 'Today'
+    unitId: today
```

```suggestion
    value: 'Today at {time}'
    unitId: today #change the key
```
any possibility to parametrized, it's okay to duplicate rather than text concatenation. cz translators don't know the context

---

Kept `Today` as its required for another lang string. added new lang string as suggested